roles/ceph-mon/tasks/start_monitor.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: activate monitor with upstart
+  file: path=/var/lib/ceph/mon/ceph-{{ ansible_hostname }}/{{ item }}
+        state=touch
+        owner=root
+        group=root
+        mode=0600
+  with_items:
+    - done
+    - upstart
+  when: ansible_distribution == "Ubuntu"
+  changed_when: false
+
+- name: start and add that the monitor service to the init sequence (ubuntu)
+  service: name=ceph-mon
+           state=started
+           enabled=yes
+           args="id={{ ansible_hostname }}"
```

Comment:
@craigtracey 
It does something similar to this:
```
root@ceph4:~# service ceph-mon start id=ceph4
start: Job is already running: ceph-mon (ceph/ceph4)
```

This can also be another option, but not sure if the service module with take the "enabled" effect for ceph-all.
```
root@ceph4:~# service ceph-all start
start: Job is already running: ceph-all
```